aries-site/src/examples/components/table/TruncatedText.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'grommet';
+
+export const TruncatedText = ({ value }) => {
+  const ref = useRef();
+  const [tip, setTip] = useState();
+
+  useEffect(() => {
+    if (ref.current && ref.current.scrollWidth > ref.current.offsetWidth)
```

Comment:
Should this also drop the tip if not needed at some point?

Reply:
Good thought. I'll update the condition